Microsoft.Toolkit.Uwp.UI.Controls/GridSplitter/GridSplitter.Events.cs

```diff
@@ -60,23 +54,32 @@ private void GridSplitter_Loaded(object sender, RoutedEventArgs e)
             _hoverWrapper = hoverWrapper;
         }
 
-        private void Gripper_KeyDown(object sender, KeyRoutedEventArgs e)
+        private void CreateGripperDisplay()
         {
-            var gripper = sender as GridSplitterGripper;
-
-            if (gripper == null)
+            if (_gripperDisplay == null)
```

Comment:
The Loaded event can fire multiple times. It is called each time a control is (re-)added to the visual tree. For example if you use a Frame with page caching and navigate to a different page, the old page content will get Unloaded event and then is put into cache. When moving back to the old page, it is re-added to the visual tree and so it will get Loaded event again, for the second time. In that case, our _gripperDisplay already exists, no need for us to re-create it.